dotCMS/src/main/java/com/dotcms/experiments/business/web/SelectedExperiment.java

```diff
@@ -0,0 +1,64 @@
+package com.dotcms.experiments.business.web;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.io.Serializable;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This is a {@link com.dotcms.experiments.model.Experiment} selected to a User, this mean this user
+ * is going to be into this {@link com.dotcms.experiments.model.Experiment}.
+ * 
+ * @see ExperimentWebAPI#isUserIncluded(HttpServletRequest, HttpServletResponse)
+ * @see com.dotcms.experiments.model.Experiment
+ */
+public class SelectedExperiment implements Serializable {
+
+    /**
+     * Experiment's name
+     */
+    @JsonProperty("name")
+    private String name;
+
+    /**
+     * Experiment's ID
+     */
+    @JsonProperty("id")
+    private String id;
+
+    /**
+     * Experiment;s page URL
+     */
+    @JsonProperty("pageUrl")
+    private String pageUrl;
```

Comment:
set all of them to final

Reply:
done https://github.com/dotCMS/core/pull/23334/commits/088b10e51e9d5864fb510bd1a7cc17241899e47e#diff-282d55898ddcc13745c364b2a7277eea0babbd5ab656e4100748e1a886ece800R21